src/qualtrics.py

```diff
@@ -0,0 +1,95 @@
+"""
```

Comment:
Might be better to put this in the data-analysis folder given this is a helper for data-analysis ?